cc_licenses/templates/includes/footer.html

```diff
@@ -1,4 +1,4 @@
-{% load bidi %}
```

Comment:
Just a note on the PR that we should check with @TimidRobot . I'm not sure if this is going to remain being the footer. I think @TimidRobot is working on another version.

Reply:
Good point. I'm just trying to make the footer look right until we get there.